scripts/generate_docs.sh

```diff
@@ -58,6 +58,7 @@ jazzy_args=(--clean
             --author_url https://github.com/apple/swift-nio
             --github_url https://github.com/apple/swift-nio
             --theme fullwidth
+            --github-file-prefix https://github.com/apple/swift-nio/tree/$version \
```

Comment:
last backslash redundant?

Reply:
Oh yes, sorry didn’t notice this was a bash array. Will fix when I’m at a computer again.